packages/embed/src/utils/setup-google-analytics-instance.spec.ts

```diff
@@ -54,7 +54,6 @@ describe('setup-google-analytics-instance', () => {
           makeTracker('UA-tracking-3', 'one.more.client.identifier'),
         ]),
     }
-    window['GoogleAnalyticsObject'] = 'ga'
     window['ga'] = GoogleAnalyticsObjectMock
```

Comment:
I think we can delete this line too:
```suggestion
```

Reply:
I think we still need it, as I guess you still want to support analytics' ga until July?